hack/run_test

```diff
@@ -311,6 +311,52 @@ function setup_replication_cluster() {
   done
 }
 
+function run_failover_test {
+  echo "Testing failover"
+  local cluster_args="-e POSTGRESQL_ADMIN_PASSWORD=pass -e POSTGRESQL_MASTER_USER=$PGUSER -e POSTGRESQL_MASTER_PASSWORD=$PASS"
+  local cid_suffix="failover"
+
+  # Setup the cluster
+  setup_replication_cluster
+
+  # Kill the master
```

Comment:
Also, I would consider `docker kill` to really shoot the postgres server, `docker stop` rather carefully stops the server.